### ISSUE TITLE
shp2json error in node v4.2.6

### ISSUE THREAD

**Issue by lestcape** [mbostock/shapefile#38]
I see you have an example that use exactly what i want to have:
https://gist.github.com/mbostock/5557726

My node version: 
```
node --version
```
v4.2.6
```
shp2json --version
```
0.6.4

v4.2.6 and 0.6.4 are the default for the ubuntu  LTS realease.

```
lsb_release -a
```
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial

Here what i do:
```
curl -o estados.zip http://mapserver.inegi.org.mx/MGN/mge2010v5_0.zip
curl -o  municipios.zip http://mapserver.inegi.org.mx/MGN/mgm2010v5_0.zip
unzip estados.zip 
unzip municipios.zip
```

```
ogr2ogr states.shp Entidades_2010_5.shp -t_srs "+proj=longlat +ellps=WGS84 +no_defs +towgs84=0,0,0"
ogr2ogr municipalities.shp Municipios_2010_5.shp -t_srs "+proj=longlat +ellps=WGS84 +no_defs +towgs84=0,0,0"
```

shp2json fail in both cases:

`shp2json states.shp -o states2.json`

(node) Buffer.set is deprecated. Use array indexes instead.
error: Invalid array length

`shp2json municipalities.shp -o municipalities2.json`

(node) Buffer.set is deprecated. Use array indexes instead.
error: Offset is outside the bounds of the DataView

Thanks.

**Comment by mbostock** [comment]
Thank you. It appears the current version requires Node v6+ as it depends on *buffer*.set using the TypedArray implementation; see the [release notes](https://github.com/nodejs/node/wiki/Breaking-changes-between-v5-and-v6). However I can make a small change to make this work on older versions.

**Comment by mbostock** [comment]
Fixed in 0.6.5.

**Comment by lestcape** [comment]
@mbostock thanks to you, you are like a machine creating useful things.... :)
